todo_app/app.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.session_items import add_item, get_item, get_items, remove_item, save_item
+from todo_app.data.trello_items import Item, add_item, get_items, update_item, remove_item
```

Comment:
Nice, that this is such a small change in this file is a good sign you've got a nice abstraction layer 😄